pkg/descheduler/strategies/pod_antiaffinity_test.go

```diff
@@ -123,6 +124,13 @@ func TestPodAntiAffinity(t *testing.T) {
 			nodes:                   []*v1.Node{node1},
 			expectedEvictedPodCount: 3,
 		},
+		{
+			description:                    "Maximum pods to evict (maxNoOfPodsToEvictPerNamespace=3)- 3",
```

Comment:
"Maximum pods to evict (maxNoOfPodsToEvictPerNamespace=3)- 3" -> "Maximum pods to evict (maxPodsToEvictPerNamespace=3) - 3",

Reply:
Fix done ~~ Please check :)